### DISCUSSION TITLE
Error importing oppenZepelin on remix.ide

### DISCUSSION THREAD

**Discussion by ChinonsoUbanna** [Cyfrin/foundry-full-course-cu#3097] [Q&A]
pls when i compile the file using remix.ide i get Error: not found https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol  everytime, i have tried many times but keep getting this

**Comment by EngrPips** [comment]
Please share your code So that I can try on my end.

**Reply by ChinonsoUbanna** [reply]
pragma solidity ^0.8.0;

// Import ERC20 Contract from OpenZeppelin
import "https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol";

// contract x { ... }
contract LW3Token is ERC20 {
  constructor(string memory _name, string memory _symbol) ERC20(_name, _symbol) {
    _mint(msg.sender, 10 * 10 ** 18);
    }
}

**Comment by ChinonsoUbanna** [comment]
pragma solidity ^0.8.0;
// Import ERC20 Contract from OpenZeppelin
import "https://t.co/4n9FefKi59";

// contract x { ... }
contract LW3Token is ERC20 {
constructor(string memory _name, string memory _symbol) ERC20(_name,
_symbol) {
_mint(msg.sender, 10 * 10 ** 18);
}
}

**Comment by ChinonsoUbanna** [comment]
I meant import "
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts%2Ftoken%2FERC20%2FERC20.sol
"

**Reply by EngrPips** [reply]
So, I tried on my end and ran into the same issue. I asked Remix Ai, and the response is below.

![Screenshot 2024-12-24 at 20 59 05](https://github.com/user-attachments/assets/c91f3976-8419-4cd1-bbf2-e32d41ef331e)

**Reply by EngrPips** [reply]
Consider developing your contract locally if you are having issues importing stuff on `Remix`
